src/test/groovy/graphql/execution/instrumentation/dataloader/DataLoaderHangingTest.groovy

```diff
@@ -304,7 +304,7 @@ class DataLoaderHangingTest extends Specification {
         @Override
         Object get(DataFetchingEnvironment environment) {
             Person source = environment.getSource()
-            DataLoaderRegistry dlRegistry = environment.getContext()
+            DataLoaderRegistry dlRegistry = environment.getGraphQlContext()
```

Comment:
How can this work??? a "context" could have been a `DataLoaderRegistry` but a GraphqlContext never is!??

Reply:
`nvironment.getGraphQlContext().get("registry")` ??

Reply:
Nope I was wrong, it should not work